Core/NoOpRunner.Core/Shapes/Player.cs

```diff
@@ -108,5 +127,25 @@ public void HandleKeyPress(KeyPress key, WindowPixel[,] gameScreen)
                     return;
             }
         }
+
+        public bool StateHasChanged()
+        {
+            return StateMachine.StateHasChanged;
+        }
+
+        public Uri GetStateAnimationUri()
+        {
+            return StateMachine.GetStatusUri();
+        }
+
+        public bool IsPlayerTurning()
+        {
+            return StateMachine.IsTurning;
+        }
+
+        public bool IsLookingLeft()
+        {
+            return StateMachine.IsTurnedLeft;
+        }
```

Comment:
These should probably be properties, not methods (`public bool StateHasChanged => StateMachine.StateHasChanged;` etc)